install/primehub-support

```diff
@@ -0,0 +1,407 @@
+#!/bin/bash
+# InfuseAI support data collection script
+HOME=$HOME
+LOG_DIR=$HOME/infuseai_support_logs_$(date +'%Y%m%d%H%M')
```

Comment:
I think we can store the log in some folder. Not in home dir.
EX: 
```LOG_DIR=$HOME/support/infuseai_support_logs_$(date +'%Y%m%d%H%M')```
or
```LOG_DIR=$HOME/log/infuseai_support_logs_$(date +'%Y%m%d%H%M')```

Reply:
That's good advice, cleaner home folder is what we are looking for!
I will create a parent folder for logs to be stored.